tests/Tasks/UpdatePackageInfoTest.php

```diff
@@ -24,6 +25,16 @@ public static function setUpBeforeClass(): void
         MetaData::$isRunningUnitTests = true;
     }
 
+    protected function setUp(): void
+    {
+        parent::setUp();
+        // Remove extension that will connect to github
```

Comment:
I think we've we've been constantly live fetching stuff from github this whole time